memor/response.py

```diff
@@ -208,6 +223,7 @@ def _validate_extract_json(json_object: Union[str, Dict[str, Any]]) -> Dict[str,
             result["tokens"] = loaded_obj.get("tokens", None)
             result["inference_time"] = loaded_obj.get("inference_time", None)
             result["model"] = loaded_obj["model"]
+            result["gpu"] = loaded_obj["gpu"]
```

Comment:
We should load all new parameters using `.get` with default values, similar to what we did for `top_k` and `top_p`.